agent/src/main/java/com/cloud/agent/properties/AgentProperties.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2021 The Apache Software Foundation.
```

Comment:
@GutoVeronezi can you please remove these Copyright lines on the source code header?